### PR TITLE
CLDR-15071 Add SIL likely data

### PR DESCRIPTION
**Pull request by macchiati**
CLDR-15071

This adds additional items to supplemental/likelySubtags.xml, without changing the existing items.

I split followup work into a separate ticket https://unicode-org.atlassian.net/browse/CLDR-16363 

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by DavidLRowe** [comment]
I don't know Java, so I'll leave it to @srl295 to review.

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+
+    public static void check(LstrType lstrType, String lang, Map<LstrType, Status> errors) {
+        final Status status = validity.getCodeToStatus(lstrType).get(lang);
+        if (status != Status.regular) {
+            errors.put(lstrType, status);
+        }
+    }
+
+    public static void main(String[] args) {
+        list("de en es es-419 fr hr it nl pl pt-BR pt-PT vi tr ru ar th ko zh-CN zh-TW ja ach af ak az ban su xx-bork bs br ca ceb cs sn co ht cy da yo et xx-elmer eo eu ee tl fil fo fy gaa ga gd gl gn xx-hacker ha haw bem ig rn id ia xh zu is jw rw sw tlh kg mfe kri la lv to lt ln loz lua lg hu mg mt mi ms pcm no nn nso ny uz oc om xx-pirate ro rm qu nyn crs sq sk sl so st sr-ME sr-Latn fi sv tn tum tk tw wo el be bg ky kk mk mn sr tt tg uk ka hy yi iw ug ur ps sd fa ckb ti am ne mr hi bn pa gu or ta te kn ml si lo my km chr");
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java:95]: move constant to top and document
(ideally in a data file if this is going to be rerun)

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+                    System.out.println(key + "\t" + value + "\t" + infoFields(value));
+                }
+                System.out.println();
+            }
+        }
+            );
+
+    }
+
+    static ImmutableMap<String, String> remap = ImmutableMap.of("iw", "he", "jw", "jv");
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java:155]: same

Comment (srl295) [reply tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java:155]: how does this relate to LIKELY_SPECIALS?

**Review by srl295 — DISMISSED** [review]
promising.

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java (hunk 2 above)
Comment (macchiati) [reply tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java:220]: Will check later.

Comment (macchiati) [reply tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java:238]: That will be fixed in later integration, https://unicode-org.atlassian.net/browse/CLDR-16363

**Comment by macchiati** [comment]
If anyone is around this weekend, I resolved the test problems, so this is ready to review.

**Comment by macchiati** [comment]
FYI, if you approve this, feel free to also merge.

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+        if (isOk(lang, script, region, errors)) {
+            add(result, source, lang, script, region, reference);
+        } else {
+            showSkip("Skipping scope, SIL", source, ltpFull.toString(), errors);
+        }
+    }
+
+    private static Multimap<String, String> readWikidata(Set<String> alreadyLangs) {
+        Multimap<String, String> result = TreeMultimap.create();
+        Path path = Paths.get(CLDRPaths.BIRTH_DATA_DIR, "/../external/wididata_lang_region.tsv");
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java:410]:
```suggestion
        Path path = Paths.get(CLDRPaths.BIRTH_DATA_DIR, "/../external/wikidata_lang_region.tsv");
```

File: tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java [hunk 4]
```diff
@@ -0,0 +1,445 @@
...
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.LinkedHashMultimap;
+import com.google.common.collect.Multimap;
+import com.google.common.collect.TreeMultimap;
+import com.ibm.icu.impl.Row;
+import com.ibm.icu.lang.UScript;
+import com.ibm.icu.text.UnicodeSet;
+import com.ibm.icu.util.Output;
+
+public class GenerateAdditionalLikely {
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateAdditionalLikely.java:41]:
```suggestion
@CLDRTool(alias = "generate-additional-likely",
description = "Generate additional likelysubtags entries from Wikidata and SIL langtags.json")
public class GenerateAdditionalLikely {
```

**Review by srl295 — APPROVED** [review]
LGTM with minor suggestions, could be a follow PR

**Comment by srl295** [comment]
OK to merge, and can followup in CLDR-16363

**Pull request merged**